pkg/cloud/aks/aks.go

```diff
@@ -0,0 +1,190 @@
+package aks
+
+import (
+	b64 "encoding/base64"
+	"encoding/json"
+	"github.com/jenkins-x/jx/pkg/log"
+	"github.com/jenkins-x/jx/pkg/util"
+	"strings"
+)
+
+var runner util.Commander

```

Comment:
I would prefer that we do not use global variable. It's better to define a type and methods if you want to keep some variable state between the executions: 

```Go
type AzRunner struct {
  runner util.Commander
}
func NewAzRunner(commander util.Commander) ut {
      return &AzRunner{..}
}

func (az *AzRunner) GetClusterClient(..)

func (az *AzRunner) azureCli(){
   az.runner.SetName(..)
}
```